src/app.js

```diff
@@ -247,9 +254,9 @@ class Messenger extends EventEmitter {
       attachments
     } = message;
 
-    if (message.is_echo) {
-      // Requires enabling `message_echoes` in your webhook, which is not the default
-      // https://developers.facebook.com/docs/messenger-platform/webhook-reference#setup
+    if (this.options.ignoreEcho && message.is_echo) {
+      // Echo messages are ordinary messages with `is_echo` set. They're only
+      // interesting if your bot wants to listen to other bots.
```

Comment:
I would have thought from the description that this would emit a `message.echo` event. Is there an alternate approach to having a bot listening to another bot? Or is this coming with:

> I can open up the logger as an example bot